src/component/movies/MovieFeedback.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { addFeedback } from '../../store/actions/MovieFeedbackActions';
+import '../../styles/css/movies/movie_feedback.css'
+
+class MovieFeedback extends Component {
+  
+    addFeedback = (feedback) => {
+        if (this.props.movie.user_feedback !== null) {
+            return;
+        }
+        this.props.addFeedback({movie: this.props.movie.id, feedback: feedback});
+    }
```

Comment:
ovdje predpostavljam da nece radi unlike, znaci kada korisnik ostavi da mu se nesto svidja i onda kasnije hoce da povuce ?

Reply:
addFeedback moze biti `addLike` pa u zavisnosti od parametra bice ili dislike ili like ili neutralan, odnosno ako ponisti odredjenu akciju (like/dislike)

Reply:
ovo mozes odraditi preko konstanti ili mozes ici sa brojevima, tipa +1 , 0 , -1

Reply:
<img width="352" alt="Screen Shot 2021-01-21 at 11 01 39" src="https://user-images.githubusercontent.com/44477868/105335224-13e88080-5bd8-11eb-8e00-de74810b7599.png">

Ja uopste nisam ni realizovao unlike opciju. To sam radio jer sam ovo gore na slici protumacio tako, dakle da korisnik moze da da like ili dislike jednom i to je to. 

Sustinski je meni jasno da svaki iole normalan sajt treba da da mogucnost i unlike-a, ali sam se ovde vodio time sto pise u opisu task-a. Moguce da ni nisam dobro protumacio.